utils/utils.c

```diff
@@ -698,7 +698,9 @@ gbRunnerGetOutput(char *cmd)
   GB_FREE(buf);
 
  out:
-  pclose(fp);
+  if (fp) {
+    pclose(fp);
+  }
```

Comment:
IMO, this is make no much sense, but no harm. It is up to you.

Because the fp = fopen(), will set the fp to NULL or the actual handler.

Reply:
Coverity reports it :-)

Reply:
Okay.